pkgs/servers/web-apps/cryptpad/default.nix

```diff
@@ -0,0 +1,63 @@
+{ lib, buildNpmPackage, fetchFromGitHub, nodejs, nixosTests }:
+
+buildNpmPackage rec {
+  pname = "cryptpad";
+  version = "5.4.0";
```

Comment:
We may want to bump this to 5.5.0 or later when this PR is ready.

Reply:
Definitely yes